RecoLocalMuon/DTSegment/src/DTSegmentUpdator.cc

```diff
@@ -122,16 +125,16 @@ void DTSegmentUpdator::fit(DTRecSegment4D* seg, bool allow3par)  const {
     if(seg->hasZed()) {
 
       // fit in-time Phi segments with the 2par fit and out-of-time segments with the 3par fit
-      if (fabs(seg->phiSegment()->t0())<40.) {
-        fit(seg->phiSegment(),allow3par,0);
```

Comment:
this part "allow3par, block3par" had block3par change from 1 in #3724 (~original version) to 0 in #4018, to now back to 1 here.
Is this just empirical (possibly riding a random variation wave) or is there reason behind each time this being an improvement?